lockup/src/types.rs

```diff
@@ -165,3 +184,13 @@ pub struct TerminationInformation {
 /// The result of the transfer poll.
 /// Contains The timestamp when the proposal was voted in.
 pub type PollResult = Option<WrappedTimestamp>;
+
+pub fn hash_vesting_schedule(vesting_schedule: &VestingSchedule, salt: &[u8]) -> Hash {
```

Comment:
I would suggest defining `VestingScheduleWithSalt` structure and serialize it as borsh.

Reply:
done